ecosystem/typescript/sdk/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ All notable changes to the Aptos Node SDK will be captured in this file. This ch
 
 N/A
 
+## 1.3.18 (2022-11-15)
+- support token property map deseralizer to read the property map in the original data format.
+
```

Comment:
Can we actually move this line 9. Because the release date will change when we actually release this.

Reply:
updated